WATOLINK/bci-gaming
# Cutscenes

## ClickUp ticket link

[Adding Cutscenes](https://app.clickup.com/t/8688q8qwa)


## Implementation description
Implemented Dialogue system + Cutscene sequence controller (both dialogue and animations) + Scene transition 
All scripts could be found under Assets/Scripts/Game/UI/CutsceneSystem.
All game objects could be found in the inspector under Cutscenes. The scene transition is a prefab.



## Steps to test
1. Open the Cutscene scene under W24 folder in Assets/Scenes
2. Start the game and hit enter until all the dialogue and animations are finished playing
3. The Cutscene scene should transition to the VideoDemo scene



## What should reviewers focus on?
Ensure all dialogue and animations are working correctly


## Checklist
